### PR TITLE
Feature/recommendation ~placements~ pods

### PR DESCRIPTION
**Pull request by sblaurock**
- Recommendations endpoint changes to utilize pod identifier in favor of declaring specific method.
- `results` becomes `numResults`.

*Do not merge* - blocked until backend `/pods/` endpoint has been deployed.

### CODE REVIEW

File: spec/src/modules/recommendations.js [hunk 1]
```diff
@@ -126,7 +127,10 @@ describe('ConstructorIO - Recommendations', () => {
         fetch: fetchSpy,
       });
 
-      recommendations.getAlternativeItems(itemId, { results }).then((res) => {
+      recommendations.getRecommendations(placement, {
+        itemIds,
+        results,
```
Comment (jjl014) [inline spec/src/modules/recommendations.js]: saw this and was immediately confused by `results`, had to look at the code to remember that it was `numResults`...

we had a discussion on this before but i don't remember why we didn't go with `numResults` 🤔

Comment (rubikzube) [reply spec/src/modules/recommendations.js]: Good point.  We should change to `numResults`.

File: README.md [hunk 2]
```diff
@@ -88,38 +88,11 @@ constructorio.autocomplete.getResults('dogs', {
 
 ### Recommendations
 
-The recommendations module can be used to retrieve item recommendation results. Responses will be delivered via a Promise. The `parameters` object is optional. Item id's may be of type string or array.
+The recommendations module can be used to retrieve item recommendation results for a supplied pod identifier. Responses will be delivered via a Promise. The `parameters` object is optional. Item id's may be of type string or array.
```
Comment (rubikzube) [inline README.md]: I would change `item recommendations` here to just be `recommendations`.  Being less descriptive might be better for future compatibility.

Comment (rubikzube) [reply README.md]: Might change `supplied pod identifier` to `given pod`

File: spec/src/modules/recommendations.js (hunk 1 above)
Comment (sblaurock) [reply spec/src/modules/recommendations.js]: 👍 I'll open a separate PR to change this for autocomplete as well.

**Comment by rubikzube** [comment]
Looks great overall, let's not merge until tomorrow

File: README.md (hunk 2 above)
Comment (sblaurock) [reply README.md]: 👍

**Pull request merged**